### DISCUSSION TITLE
이번 깃헙 특강 어땠는가

### DISCUSSION THREAD

**Discussion by Dangtae** [happyCoVid-19/PS#4] [Polls]
.

**Comment by minrongtic** [comment]
정말 좋았음 

- GUI가 CLI보다 더 많은 기능을 제공하는 줄 몰랐음 
-> 보통..학교에서는 CLI 안쓰면 죽는 분위기라 GUI 써볼 생각을 안했는데 Git Graph 정말 편리한 것 같음 
- git graph, snipaste, chatgpt desktop ver, etc 각종 꿀팁도 많이 얻을 수 있었음 
- organization repo 파서 discussion이나, issue, project, wiki 등등 써보고싶은 생각이 1주차 때부터 있었는데 git 자체에 대한 두려움 때문에 못해보다가 특강 덕분에 자신감이 생겨서 시도할 수 있게 됨. 
- figma로 강의하시는 것도 되게 신선했음. 다른 캠퍼분들의 피그마 꿀팁도 얻게 되고 좋았음. 
- 다른 것들 다 차치하고서 맨 처음 생활코딩으로 파이썬 공부했었는데, 그 목소리 그대로 이고잉님께 강의를 받는다는게 믿기지가 않았음. 진짜 연예인 보는 느낌 아니면 스타강사 현강 듣는 느낌. git 강의 자체는 몇 번 들어봤는데, 이번 강의가 압도적으로 좋았음. 천천히 이해 잘되게 여러번 반복해주시고, 직접 그 자리에서 같이 오류 해결해주시니까 더 든든했음. 완전 만족!!

**Comment by Solsol1014** [comment]
세상에 내가 몇십만 구독자 코딩 유튜버의 강의를 듣게 될줄이야 일단 여기서부터 우히힛이었음
